package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "geyser-app",
   "private": true,
-  "version": "0.10.1",
+  "version": "0.12.0",
```

Comment:
why did you go from a 0.10.1 to 0.12.0 ? :D

At max it would go to 0.11.0, but it's a pretty small update so, I'd say, 0.10.2, honestly. What do you think ? @Agill-Sheron

Reply:
Hey Sajal, in the DEPLOYMENT.md the command to update the app version is the following: `yarn version [ minor | major | patch ]` I thought that `minor` would be the smallest step up 😅 but I definitely agree with you. 

Is it okay to manually update the version in the `package.son`?

Reply:
Actually this has already been done, for this release. We can close this PR.